android-components/components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -107,6 +107,30 @@
 
         </LinearLayout>
 
+        <LinearLayout
+            android:id="@+id/add_on_status_blocklisted"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:orientation="vertical">
```

Comment:
To avoid flickering in the UI (Showing and hiding), we could make this view invisible (`GONE`) by default and just make it visible when it's needed.